lib/xlsx/xlsx.js

```diff
@@ -108,7 +108,7 @@ XLSX.prototype = {
       sharedStrings: model.sharedStrings,
       media: model.media,
       mediaIndex: model.mediaIndex,
-      date1904: model.properties.date1904,
+      date1904: model.properties && model.properties.date1904,
```

Comment:
I'm not sure that the meaning of this line of code, but I think that you want to use || operator instead of &&. 
Are you sure in this line? Can you provide me little clarification for it? :)